proxyd/proxyd.go

```diff
@@ -350,6 +351,15 @@ func Start(config *Config) (*Server, func(), error) {
 				copts = append(copts, WithPollerInterval(time.Duration(bgcfg.ConsensusPollerInterval)))
 			}
 
+			for _, be := range bgcfg.Backends {
+				if fallback, ok := bgcfg.FallbackBackends[be]; !ok {
+					log.Crit("Error Backend Not Found in Backend Fallback Configurations", "Fallback Name", be)
```

Comment:
lower case logging
"Fallback Name" should be "fallback_name" so we can to structured log search